frontend/src/app/accountCreation/PasswordForm/PasswordForm.tsx

```diff
@@ -48,9 +48,9 @@ export const PasswordForm = () => {
         hints.push("at least 8 characters");
       }
       if (hints.length === 1) {
-        return "Your password must have " + hints[0];
+        return "Your password must be " + hints[0];
       }
-      return `Your password must have ${hints.slice(0, -1).join(", ")} and ${
+      return `Your password must be ${hints.slice(0, -1).join(", ")} and ${
         hints[hints.length - 1]
```

Comment:
Can we update the error message commas so that there's a comma on the second to last item when there's 3 or more? @ClydeDroid @emmastephenson

Reply:
I think it would be something like 

```
if (hints.length === 1) {
        return "Your password must have " + hints[0];
      }
else if (hints.length >= 3) {
return `Your password must have ${hints.slice(0, -1).join(", ")}, and ${
        hints[hints.length - 1]
      }`;
} else {
      return `Your password must have ${hints.slice(0, -1).join(", ")} and ${
        hints[hints.length - 1]
      }`;
}

```